src/main.rs

```diff
@@ -62,10 +63,22 @@ where
     result.entry.is_some()
 }
 
+/// Checks whether we have any certificates installed in certstore. This is used
+/// to detect whether Fobnail has been provisioned.
+fn have_certchain<T>(trussed: &mut T) -> bool
```

Comment:
According to today's discussion we probably won't keep whole chain saved, only certificates that will be created in 4.5. This function may work without changes to its code, but it should be renamed to something more appropriate.